index.bs

```diff
@@ -324,6 +324,11 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     that this does not constitute [=user verification=] because [=TUP=], by definition, is not capable of [=biometric
     recognition=], nor does it involve the presentation of a shared secret such as a password or PIN.
 
+: <dfn>Resident Key</dfn>
+: <dfn>RK</dfn>
+:: A [=resident key=] is a [=credential private key=] that can be found and used by the authenticator based on only the 
+    associated RP ID. 
+
```

Comment:
Normally I don't care about renaming. But in this case, I just cannot see any merit to the  name "requireClientSideResidentPrivateKey". Neither the name "ClientSideResidentPrivateKey" nor the name "ResidentKey" means anything to the developer. The developer will read documentation to understand what the method does anyway, just like attachment or timeout. The name should be a hint of the behavior, not a word-by-word description of the behavior of the API. I am perfectly happy with adding more description to the name resident key or add a description that equates "client-side credential private key". In addition, no web developer reads WhatWG/W3C spec. Using fully qualified term such as credential private key or client-side doesn't mean anything to developers. I'd love to spare developers the pain of writing a 35-characters name every time they wants to use a feature.